src/V3ExecGraph.cpp

```diff
@@ -380,12 +385,14 @@ class PackThreads final {
                 // If no tasks were added to the normal thread schedule, remove it.
                 if (schedule.mtaskState.empty()) result.erase(result.begin());
                 result.emplace_back(ThreadSchedule{size});
+                std::fill(busyUntil.begin(), busyUntil.end(), schedule.endTime());
                 continue;
             }
 
             if (!bestMtaskp && mode == SchedulingMode::WIDE_TASK_SCHEDULING) {
                 mode = SchedulingMode::SCHEDULING;
                 if (!schedule.mtaskState.empty()) result.emplace_back(ThreadSchedule{m_nThreads});
+                std::fill(busyUntil.begin(), busyUntil.end(), schedule.endTime());
```

Comment:
If no task was added, (how is that possibly by the way, you won't enter wide mode if no wide task is ready?), what is the value of `schedule.endTime()` ?

Reply:
Same as above.